src/scrabble.js

```diff
@@ -1,6 +1,55 @@
+const letters = {
+  value1: ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T" ,'a', 'e', 'i', 'o', 'u', 'l', 'n', 'r', 's', 't'],
+  value2: ["D", "G", "d", "g"],
+  value3: ["B", "C", "M", "P", "b" ,"c" ,"m", "p"],
+  value4: ["F", "H", "V", "W", "Y", "f", "h", "v", "w", "y"],
+  value5: ["K", "k"],
+  value8: ["J", "X" ,"j" ,"x"],
+  value10: ["Q", "Z", "q", "z"]
+}
 class Scrabble {
-  // Write your implementation here
+  // Write your implementation here#
+  constructor(word){
+    this.word = word
+  }
 
-}
+  score(){
 
+    if(this.word === ""){
+      return 0
+    }
+    if(this.word === null){
+      return 0
+    }
+    if(this.word === " \t\n"){
+      return 0
+    }
+    let result = 0
+    for(let i = 0; i < this.word.length; i++){
+      if(letters.value1.includes(this.word[i])){
+        result = result + 1
```

Comment:
you could change `result = result + 1` to `result += 1` and it would produce the same result. but if you find it easier the way you did it no need to change it 👍